frontend/src/components/Calendar/Calendar.tsx

```diff
@@ -80,15 +79,11 @@ export const Calendar = (props: { selectedFacilityId: number }) => {
                 <>
```

Comment:
can consider putting `<div key={startingMonth}>` instead of empty tag to get rid of the unique keys error

Reply:
Resolved in a55dbb9

Thanks Marcus! :smile: